io-sim/io-sim.cabal

```diff
@@ -77,14 +77,14 @@ library
                        ScopedTypeVariables,
                        TypeFamilies
   build-depends:       base              >=4.9 && <4.19,
-                       io-classes       ^>=1.0,
+                       io-classes       ^>=1.1,
```

Comment:
Would `>= 1.0 && < 1.2` also suffice? I think building with `1.0` would cause some deprecation warnings to be emitted with respect to moving the `MonadMVar` module, but would it lead to actual build failures?